src/index.test.ts

```diff
@@ -308,3 +308,51 @@ describe('i18n', () => {
         expect(logger.log).toHaveBeenCalledTimes(callsLength + 1);
     });
 });
+
+const logger = {log: jest.fn()};
+describe('Incorrect instance initialization', () => {
+    beforeEach(() => {
+        i18n = new I18N({logger});
+        jest.clearAllMocks();
+    })
+
+    test('throw while lang or keyset are not defined', () => {
```

Comment:
To be consistent with other tests let's call it like `it('should throw whie...'`